api/swagger.yml

```diff
@@ -1231,85 +1420,71 @@ paths:
         204:
           description: policy detached successfully
         401:
-          $ref: "#/responses/Unauthorized"
+          $ref: "#/components/responses/Unauthorized"
         404:
-          description: policy attachment not found
-          schema:
-            $ref: "#/definitions/error"
+          $ref: "#/components/responses/NotFound"
         default:
-          description: generic error response
-          schema:
-            $ref: "#/definitions/error"
+          $ref: "#/components/responses/ServerError"
 
   /repositories:
     get:
       tags:
         - repositories
       parameters:
-        - in: query
-          name: after
-          type: string
-        - in: query
-          name: amount
-          type: integer
-          default: 100
+        - $ref: '#/components/parameters/PaginationAfter'
+        - $ref: '#/components/parameters/PaginationAmount'
       operationId: listRepositories
       summary: list repositories
       responses:
         200:
           description: repository list
-          schema:
-            type: object
-            properties:
-              pagination:
-                $ref: "#/definitions/pagination"
-              results:
-                type: array
-                items:
-                  $ref: "#/definitions/repository"
-        401:
-          $ref: "#/responses/Unauthorized"
-        default:
-          description: generic error response
-          schema:
-            $ref: "#/definitions/error"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RepositoryList'
```

Comment:
The mix of single and double quotes is confusing.  The difference (https://yaml.org/spec/1.2/spec.html#style/flow/single-quoted, https://yaml.org/spec/1.2/spec.html#style/flow/double-quoted is in how they handle escaping.  But here we don't need _any_ quoting AFAICT; using both is strange.

Reply:
Converted everything to double-quotes